Makefile

```
@@ -8,13 +8,13 @@ help: ## Show this help
 	@grep -E '^[a-zA-Z\.\-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
 .PHONY: paas
-paas:
+paas:  ## Sets the paas environment variables for over all PAAS deployment
```

I intentionally left this `paas` task undocumented as it's not useful on its own (all the individual environment targets have `paas` as a dependency). With the `##` comment here it ends up being listed in `make help` which is a little confusing if you've not used the `Makefile` before.

We could have something like this, but I would question the benefit of a comment, over for example changing the target to be more descriptive.

```suggestion
paas:  # Set the PaaS platform variables
```

A better option might:

```suggestion
set-paas-platform:
```